test/foundry/IPAccountStorage.t.sol

```diff
@@ -2,14 +2,18 @@
 pragma solidity ^0.8.23;
 
 import { IIPAccount } from "../../contracts/interfaces/IIPAccount.sol";
+import { BaseModule } from "../../contracts/modules/BaseModule.sol";
+import { Errors } from "../../contracts/lib/Errors.sol";
 
 import { MockModule } from "./mocks/module/MockModule.sol";
 import { BaseTest } from "./utils/BaseTest.t.sol";
 
-contract IPAccountStorageTest is BaseTest {
+contract IPAccountStorageTest is BaseTest, BaseModule {
```

Comment:
Why inherit BaseModule for a test?

Reply:
To simulate registered module access the ipaccout.